docs/docs/data-sources/amazonses.md

```diff
@@ -18,17 +22,25 @@ ToolJet requires the following to connect to Amazon SES:
 
 It is recommended to create a new IAM user for the database so that you can control the access levels of ToolJet.
 
-Click on the 'Save' button to save the data source.
+Click on the **Save** button to save the data source.
 
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/datasource-reference/amazonses/connection.png" alt="Amazon SES" width="600" />
+<img style={{ border:'0', marginBottom:'15px' }} className="screenshot-full" src="/img/datasource-reference/amazonses/connection.png" alt="Amazon SES" />
+
+</div>
 
 </div>
 
+<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
```

Comment:
This looks a bit odd. We just have a header and sentence below with 24px padding at the top and bottom. Change this.